README.md

```diff
@@ -114,7 +114,17 @@ representing the resource name and `unit` labels representing the resource unit.
   * kube_pod_container_resource_limits_memory_bytes
   * kube_pod_container_resource_requests_nvidia_gpu_devices
   * kube_pod_container_resource_limits_nvidia_gpu_devices
-  
+* **The following non-generic resource metrics for nodes are marked deprecated. They will be removed in kube-state-metrics v2.0.0.**
```

Comment:
I think it would be good if we document with which flags one can disable all deprecated metrics today, to already have the chance to adapt everything to kube-state-metrics v2.0 state.

Reply:
Yep. Considering more and more metrics maybe deprecated, maybe we should make a decision about whether we need to add:
* separate knob
* global knob
to control the whether to scrape the old deprecated metrics.

The first option will give end users more fine control on what deprecated metrics to disable and what deprecated metrics should still be collected.
The second option will make the deprecation of all old metrics more easily. However, the end user control requirement above maybe not satisfied.

Reply:
Let's keep separate flags for each until someone complains, and generally as long as we document all of them it's fine in my opinion. It easily escalates when there is more than one way to do things.